src/lib/platform/EiScreen.h

```diff
@@ -141,6 +142,33 @@ class EiScreen : public PlatformScreen {
 #if HAVE_LIBPORTAL_INPUTCAPTURE
     PortalInputCapture* portal_input_capture_;
 #endif
+
+    struct HotKeyItem {
+    public:
+        HotKeyItem(std::uint32_t mask, std::uint32_t id);
+        bool operator<(const HotKeyItem& other) const { return mask_ < other.mask_; };
```

Comment:
would it be worth it to add `>` and `==` operators  ?

Reply:
I think the additional operators are necessary. For example many algorithms are implemented in terms of just `<` anyway. So if we need only `<` then it's fine.

Reply:
just ftr, this is all mostly copy-paste with modifications from the X11 implementation, so some of the decisions are just "whatever was needed over there" :)